bindata/network/ovn-kubernetes/managed/008-route.yaml

```diff
@@ -15,3 +15,6 @@ spec:
   to:
     kind: Service
     name: ovnkube-master
+{{ if .OVNSbDbRouteHost }}
+  host: {{.OVNSbDbRouteHost}}
```

Comment:
Did you see any issue by specifying route host? I remember I got permission issue when host is specified.